python/lsst/ts/wep/task/cutOutDonutsBase.py

```diff
@@ -810,6 +828,11 @@ def cutOutStamps(self, exposure, donutCatalog, defocalType, cameraName):
         stampsMetadata["EFFECTIVE"] = np.array(isEffective).astype(int)
         stampsMetadata["ENTROPY"] = np.array(stampsEntropy)
 
```

Comment:
Just run the donut fit here once. Or alternatively, just change it so that `fit_radius` is the `run` method of the task and keep the structure here as it is.